electron_app/helpers/_handleDisk.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
```

Comment:
Is it typical for helper files to start with an underscore?

Also the file is `_handleDisk.js` but the method it contains is `handleFolder`, shouldn't these names be the same?